source/form.html

```diff
@@ -0,0 +1,188 @@
+          <fieldset class="form__group">
+            <legend class="visually-hidden">Информация о коте</legend>
+            <ul class="form__list">
+              <li class="form__item">
+                <label class="form__label" for="cat-name">Имя:<sup>*</sup></label>
+                <input class="form__field" type="text" id="cat-name" name="cat-name" placeholder="Барсик" required>
+              </li>
+              <li class="form__item">
+                <label class="form__label" for="cat-weight">Вес (кг):<sup>*</sup></label>
+                <input class="form__field" type="number" id="cat-weight" name="cat-weight" placeholder="7" required>
+              </li>
+              <li class="form__item">
+                <label class="form__label" for="cat-age">Возраст (лет):</label>
+                <input class="form__field" type="number" id="cat-age" name="cat-age" placeholder="7">
+              </li>
+            </ul>
+
+          </fieldset>
+
+          <fieldset class="form__group">
+            <legend class="visually-hidden">Выбор программы</legend>
+            <ul class="form__program">
+              <li class="form__program-item">
+                <label class="control">
+                  <input class="visually-hidden control-input" type="radio" name="program" value="slimming" checked>
+                  <span class="control-mark"></span>
+                  <span class="control-label">Похудение</span>
+                </label>
+              </li>
+              <li class="form__program-item">
+                <label class="control">
+                  <input class="visually-hidden control-input" type="radio" name="program" value="weight-gain">
+                  <span class="control-mark"></span>
+                  <span class="control-label">Набор массы</span>
+                </label>
+              </li>
+              <li class="form__program-item">
+                <label class="control">
+                  <input class="visually-hidden control-input" type="radio" name="program" value="advice">
+                  <span class="control-mark"></span>
+                  <span class="control-label">Не знаю (Нужен ваш совет)</span>
+                </label>
+              </li>
+            </ul>
+          </fieldset>
+
+          <fieldset class="form__group">
+            <legend class="form__legend">Контактные данные (владельца кота)</legend>
+            <ul class="form__list">
+              <li class="form__item">
+                <label class="form__label" for="email">E-mail:<sup>*</sup></label>
+                <input class="form__field" type="email" id="email" name="email" placeholder="kuklachev@gmail.com" required>
+              </li>
+              <li class="form__item">
+                <label class="form__label" for="">Телефон:<sup>*</sup></label>
+                <input class="form__field" type="phone" id="phone" name="phone" placeholder="8 (960) 900-60-90" required>
```

Comment:
https://www.w3schools.com/html/html_form_input_types.asp
```suggestion
                <input class="form__field" type="tel" id="phone" name="phone" placeholder="8 (960) 900-60-90" required>
```